.github/ISSUE_TEMPLATE/user_story.md

```diff
@@ -21,8 +18,27 @@ assignees: ''
 
 <!-- What is the point of this user story ? Why is it necessary ? -->
 
+## Definition of ready
+
+<!-- All these items are checked by an implementation team so it can begin work -->
+<!-- DO NOT CHECK THESE YOURSELF -->
+
+- [ ] The user story is well defined
+- [ ] Acceptance criterias are defined
+- [ ] The individual responsible for validating the user story is assigned
+- [ ] The user story can be shown off
+- [ ] Does not depend on an missing feature
+- [ ] Negociable with implementors
+- [ ] Valuable to the end users
+- [ ] Estimable by the implementors
+- [ ] Small enough to fit in a sprint
+- [ ] Testable by implementors
+
```

Comment:
what about adding : 
- api contracts
- wireframes if front involved

Reply:
I don't think the API contracts have anything to do with the definition of ready: it's not something the one that writes the user story can do anything about, it's up to us